src/webui/www/private/views/about.html

```diff
@@ -676,6 +676,7 @@ <h3><a name="howto"></a><a name="SEC4" href="#TOC4">How to Apply These Terms to
             <td><span id="zlibVersion"></span></td>
         </tr>
     </table>
+    <p>QBT_TR(The free IP to Country Lite database by DB-IP is used for resolving the countries of peers. The database is licensed under the Creative Commons Attribution 4.0 International License)QBT_TR[CONTEXT=AboutDialog] (<a href="https://db-ip.com/" target="_blank" rel="noopener noreferrer">https://db-ip.com/</a>)</p>
```

Comment:
IIRC we already controlled Referer via HTTP header globally, so you should drop `noreferrer` here.

From what I read on https://mathiasbynens.github.io/rel-noopener/ the problem behind `noopener` is nasty. 
https://developers.google.com/web/tools/lighthouse/audits/noopener?hl=en#recommendations suggested that we use `rel="noopener"` when `target="_blank"` is also in use. And I think we should scan our links in WebUI files.

Reply:
Also we might be able to omit `noopener` too since web browsers are implementing countermeasures:
https://github.com/whatwg/html/pull/4330
https://bugs.chromium.org/p/chromium/issues/detail?id=898942
https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Experimental_features#Security

Reply:
Ok, `noreferrer` is dropped.